Topic3/src/org/maugiam/tdd/main/ArabicRomanConverter.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * 
+ */
+package org.maugiam.tdd.main;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+import org.maugiam.collections.BidirectionalHashMap;

```

Comment:
Warning: The BidirectionalHashMap is not par of this pull request, but we already discussed what it does.